frontend/src/components/AppDateInput.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div class="text_label">
```

Comment:
I wonder if it would be helpful to include a date format hint on the label e.g.:
![image](https://user-images.githubusercontent.com/10735511/142096124-4cee0f81-7d63-4e08-a7b2-68a99cfe4555.png)

In that example it's just a seperate <span> directly after the label with css:
```
.date-pattern {
    color: #aaaaaa;
    padding-left: 10
px
;
}
```